tests/dataframe/test_repr.py

```diff
@@ -219,12 +207,27 @@ def test_repr_with_unicode(make_df):
 </table>
 <small>(Showing first 3 of 3 rows)</small>
 </div>"""
+
+    is_parquet = "Parquet" in str(df._builder)
```

Comment:
The input data format is defined as a param in make_df https://github.com/Eventual-Inc/Daft/blob/main/tests/conftest.py#L54-L63, and I'm not sure how to access it in the test itself, so I went with this kinda hacky method for now. Will do more research tho